demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -172,12 +172,15 @@ def _search_packs_by_items_names_or_ids(items_names, items_list):
             items_names = [items_names]
 
         for item_name in items_names:
+            item_possible_machine_names = [item_name, f'incident_{item_name}', f'indicator_{item_name}',
```

Comment:
```suggestion
            item_possible_names = [item_name, f'incident_{item_name}', f'indicator_{item_name}',
```
Change it in. the usage as well